infrastructure/queues.tf

```
@@ -66,4 +66,33 @@ module "sqs-nems-queue-topic" {
       }
     ]
   })
+}
+
+module "nems_sns_queue" {
+  source             = "./modules/sns"
```

Could we add a count condition to this block? Because when I tried to run terraform with this branch and ndrd, it got an error reading value `module.sqs-nems-queue[0].endpoint`. Seems that we only have the `sqs-nems-queue` when is_sandbox is false.

```suggestion
module "nems_sns_queue" {
  count                 = local.is_sandbox ? 0 : 1
  source             = "./modules/sns"
```

---

Yep, missed this. Done.